mipdb/commands.py

```diff
@@ -30,6 +32,37 @@ def entry():
     pass
 
 
+@entry.command()
+@cl.argument("file", required=True)
+@handle_errors
+def load_folder(file):
+    dbconfig = get_db_config()
+    db = MonetDB.from_config(dbconfig)
+    for subdir, dirs, files in os.walk(file):
+        if dirs:
+            continue
+        print(f"Data model {subdir} is being loaded...")
+        reader = JsonFileReader(subdir + "/CDEsMetadata.json")
```

Comment:
You should not use string concat on paths. Use `os.path.join` or better `pathlib`.